### PR TITLE
feat: add fourbyte github integration

### LINKED ISSUE #11377: Create fallback query to decode tx input data
# Problem
 
When https://www.4byte.directory is down, it is not possible to decode the tx data.

# Implementation

I found github database that can be queries as a fallback: https://github.com/ethereum-lists/4bytes
Fallback query can be implemented to query second source in case of connection timeout

### PR DESCRIPTION
**Pull request by alaibe**
fixes https://github.com/status-im/status-desktop/issues/11377

### CODE REVIEW

**Review by stefandunca — APPROVED** [review]
Nice improvement.

File: services/wallet/thirdparty/fourbyte_github/client_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	}))
+	defer srv.Close()
+
+	fb := NewClient()
+	fb.Client = srv.Client()
+	fb.URL = srv.URL
+
+	res, err := fb.Run("0xa9059cbb000000000000000000000000e0e40d81121d41a7d85d8d2462b475074f9df5ec0000000000000000000000000000000000000000000000000000000077359400")
+	require.Nil(t, err)
+	require.Equal(t, res.Signature, "transfer(address,uint256)")
```
Comment (Cuteivist) [inline services/wallet/thirdparty/fourbyte_github/client_test.go]: Please add test for `ID`.

**Review by Cuteivist — APPROVED** [review]
Great change!

Approving, but please add single liner I requested before merge.

File: services/wallet/thirdparty/fourbytegithub/client.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		return nil, err
+	}
+	return resp, nil
+}
+
+func (c *Client) Run(data string) (*thirdparty.DataParsed, error) {
+	if len(data) < 10 || !strings.HasPrefix(data, "0x") {
+		return nil, errors.New("input is badly formatted")
+	}
+	// methodSigData := data[2:10]
```
Comment (Cuteivist) [inline services/wallet/thirdparty/fourbytegithub/client.go]: Hardcoded parameter :)

**Review by Cuteivist — CHANGES_REQUESTED** [review]

File: services/wallet/thirdparty/fourbytegithub/client_test.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	defer srv.Close()
+
+	fb := NewClient()
+	fb.Client = srv.Client()
+	fb.URL = srv.URL
+
+	res, err := fb.Run("0xa9059cbb000000000000000000000000e0e40d81121d41a7d85d8d2462b475074f9df5ec0000000000000000000000000000000000000000000000000000000077359400")
+	require.Nil(t, err)
+	require.Equal(t, res.Signature, "transfer(address,uint256)")
+	require.Equal(t, res.ID, "0x3734373236313665373336363635373232383631363436343732363537333733326337353639366537343332333533363239")
```
Comment (Cuteivist) [inline services/wallet/thirdparty/fourbytegithub/client_test.go]: Shouldn't ID be `0xa9059cbb` in this case?

See here as an example https://arbiscan.io/tx/0x2a340798c81ae08251a5cd332acad9651c8be360737d10d317607ff38d9f5e82

**Review by Cuteivist — APPROVED** [review]

**Pull request merged**